### PR TITLE
Architecture update & other Ben changes

### PR DESCRIPTION
**Pull request by ciprianprohozescu**

### CODE REVIEW

File: sections/chapters/introduction/overview.tex [hunk 1]
```diff
@@ -1,11 +1,55 @@
-As previously mention our goal is to design and develop a tool that will asssist the user in gathering insightful data from unstructured data.
-In order to achieve this target we decided to implement various methods into our project. Figure 1.1 displays the architecture designed for this specific project.  
+As previously mentioned, our goal is to design and develop a tool that will asssist the user in gathering insightful data from unstructured data.
```
Comment (BalazsAgardi) [inline sections/chapters/introduction/overview.tex]: I really hate the word "unstructured" I think unorganized works so much better. I guess it's too late to change it everwhere

File: sections/chapters/introduction/overview.tex [hunk 2]
```diff
@@ -1,11 +1,55 @@
...
+We describe the metadata in detail in section ~\ref{sec:metadata}, and the data matching library in sections
+~\ref{sec:discovery-strategy} and ~\ref{sec:the-data-matching-library}.
+
+We present implementation details in section ~\ref{sec:discovery-tool-implementation}, as a proof of concept of a system
+that can reliably make use of the algorithms included in the library.
+We include a web client in section ~\ref{sec:user-interface} that serves as both a management and display tool for the catalog.
+For example, the web client is capable of requesting random data to be generated for testing.
+Alternatively, for scenarios in which the user might need static reports instead of a full-blown web interface, we implemented
+a metadata visualizer (section ~\ref{sec:metadata-visualization}).
+
```
Comment (BalazsAgardi) [inline sections/chapters/introduction/overview.tex]: This whole section really overlaps with the conclusion. I don't necessary mind it, but we have to be okay with some duplication then.

File: sections/chapters/introduction/overview.tex (hunk 1 above)
Comment (ciprianprohozescu) [reply sections/chapters/introduction/overview.tex]: I can do a quick search

File: sections/chapters/introduction/overview.tex (hunk 2 above)
Comment (ciprianprohozescu) [reply sections/chapters/introduction/overview.tex]: Yes, it's very much supposed to overlap with the conclusion. The conclusion should reiterate ideas from the rest of the report. Also, I haven't seen the conclusion explain some of the things I do here, such as the need for 2 kinds of data, and what each section of the report does, etc.

File: sections/chapters/introduction/overview.tex (hunk 1 above)
Comment (ciprianprohozescu) [reply sections/chapters/introduction/overview.tex]: Done

**Pull request merged**